src/sketch.js

```diff
@@ -1,3 +1,28 @@
+/**
+ * CONSTANTS AND DEFINITIONS
+ */
+
+// sketch size values
+const HEIGHT = 400;
+const WIDTH = 1000;
+
+// terrain patches sizes
+const FIELD_HEIGHT = HEIGHT + 200;
+const FIELD_WIDTH = WIDTH + 800;
+const RESOLUTION = 20;
+const ROWS = FIELD_HEIGHT / RESOLUTION;
+const COLUMNS = FIELD_WIDTH / RESOLUTION;
+
+// colors
+const BACKGROUND_COLOR = [118, 168, 218];
+const TERRAIN_COLOR = [239, 84, 100, 80];
```

Comment:
Couldn't we maybe extract those in a config file and just import here?

Reply:
I think it's better to leave them here for faster iteration.

In larger projects we may come up with a default method for handling a large number of parameters.